blog/2022-05-10-phase2.mdx

```diff
@@ -6,7 +6,7 @@ author_image_url: /img/blog/elena.png
 author_description: Elena is the Founder and CEO of Iron Fish — previously worked at Microsoft and Airbnb. Fell down the cryptocurrency rabbit hole in 2017. Really didn't want her insurance to know she eats pizza.
 hide_table_of_contents: true
 title: Phase 2 Incentivized Testnet
-description: Read on about how *you* can participate in Phase 2 of the Incentivized Testnet today
+description: Read on about how *you* can participate in Phase 2 of the Incentivized Testnet today!
```

Comment:
This changes the way it's interpreted, and maybe not how elena intended. If you want to make changes like this you'll need to get elena's review.

```suggestion
description: Read on about how *you* can participate in Phase 2 of the Incentivized Testnet today
```

Reply:
Got it @NullSoldier . Just changed it back